cpp/src/bitmask/null_mask.cu

```diff
@@ -695,33 +617,46 @@ rmm::device_buffer bitmask_binop(Binop op,
   }
 
   if (masks.size() > 0) {
-    return cudf::detail::bitmask_binop(op, masks, offsets, view.num_rows(), stream, mr);
+    return cudf::detail::bitmask_binop(
+      [] __device__(bitmask_type left, bitmask_type right) { return left & right; },
+      masks,
+      offsets,
+      view.num_rows(),
+      stream,
+      mr);
   }
 
   return null_mask;
```

Comment:
Hm, we have the same problem here. If all of the table columns are non-nullable, we are returning uninitialized memory!